go/protocol/keybase1/extras.go

```diff
@@ -2000,6 +2000,19 @@ func (t TeamInvite) KeybaseUserVersion() (UserVersion, error) {
 	return ParseUserVersion(UserVersionPercentForm(t.Name))
 }
 
+// ContainsUID returns true if invite type is Keybase and uid is found
+// in the invite name. This does not mean that this is a valid invite
+// for that uid. Proper parsing using ParseUserVersion or
+// KeybaseUserVersion should be used afterwards.
+func (t TeamInvite) ContainsUID(uid UID) bool {
```

Comment:
This function is kind of imprecise. Is it necessary? Compared with just `if inviteUv, err := invite.KeybaseUserVersion(); err == nil {`

Reply:
Ah ok, there was some rough idea that ContainsUID would immediately jump into searching in invite.Name, saving some function calls and potentially resulting in better perf - since this happens for every membership × every (still) active invite. But I haven't actually benchmarked any of this and later changed ContainsUID to also check invite type/category, so it's no use now. 
Thanks for pointing this out!